### PR TITLE
refactor: align all modals

### LINKED ISSUE #1962: center the navigation in onboarding
In the recent modal update, the last step in onboarding is looking funny. I guess this happened since this is technically using the modal component.

image

### PR DESCRIPTION
**Pull request by juliendonck**
Todo:
- [x] align all modals

small fix:
- [x] closes #1962 

Signed-off-by: Julien Donck <hello@juliendonck.com>

### CODE REVIEW

**Comment by rudolfs** [comment]
There are still some typescript errors: https://buildkite.com/monadic/radicle-upstream/builds/9758#02fa4b83-8195-468e-89b8-236c78a090d2/370-377.

**Comment by juliendonck** [comment]
> There are still some typescript errors: https://buildkite.com/monadic/radicle-upstream/builds/9758#02fa4b83-8195-468e-89b8-236c78a090d2/370-377.

Fixed

File: ui/DesignSystem/Modal.svelte [hunk 1]
```diff
@@ -6,8 +6,12 @@
  LICENSE file.
 -->
 <script lang="typescript">
+  import { Emoji } from "ui/DesignSystem";
   export let dataCy: string = "";
   export let style: string = "";
+  export let emoji: string = "";
```
Comment (rudolfs) [inline ui/DesignSystem/Modal.svelte:12]: I'd prefer if we'd type props like this:

```suggestion
  export let emoji: string | undefined = undefined
```

File: ui/DesignSystem/Modal.svelte [hunk 2]
```diff
@@ -6,8 +6,12 @@
  LICENSE file.
 -->
 <script lang="typescript">
+  import { Emoji } from "ui/DesignSystem";
   export let dataCy: string = "";
   export let style: string = "";
+  export let emoji: string | undefined = undefined
+  export let title: string = "";
+  export let desc: string = "";
```
Comment (rudolfs) [inline ui/DesignSystem/Modal.svelte:14]: Would be nice to convert all of the props to the same style.

Comment (juliendonck) [reply ui/DesignSystem/Modal.svelte]: https://github.com/radicle-dev/radicle-upstream/pull/2082/commits/ffcd7be5937ebbb4cc2dbfc89d499da9641386a8

**Review by rudolfs — APPROVED** [review]
utACK

**Pull request merged**